packages/teams-js/test/private/files.spec.ts

```diff
@@ -576,7 +576,7 @@ describe('files', () => {
     it('should send the message to parent correctly', () => {
       utils.initializeWithContext('content');
 
-      const callback = jest.fn((err) => {
```

Comment:
please make sure to sync your main branch to match what's top of the repo, and run yarn. We've update prettier recently and looks like you don't have these changes because you're re-formatting the files with old patterns.

Reply:
I had synced the code in my local repo with the forked repo. Furthermore, i have ran **yarn install** followed by **yarn build** which caused the changes in the test file which is being pointed out here. As per offline conversation with you resolving this.

Reply:
To add a little more guidance here, once you sync the latest changes, `yarn install`, and build the repo, you may see a ton of files modified after the build (likely with no actual content changes). (This seems to be more prevalent on Windows machines than Macs.)
 
There are 2 options to fix this in your local repo.
 
Option 1:	
Delete your local repo and re-clone.

Option 2:
OR, from the root:
	

1. `yarn clean:full`
2. `yarn install`
3. `yarn build`
4. `git restore .` // This will undo all the changes that the build created.
5. Repeat steps 1-3.
6. No changes should exist.

Reply:
Thanks for adding detailed guidance Erin. Both the workarounds didn't help me. I tried in both Mac and Windows machines. Hence going ahead with these re-formatted changes to the files.spec.ts file with old patterns. Let me know if i am still missing something here.